backend/api/av.json

```diff
@@ -0,0 +1,6 @@
+{
+  "Av": {
+    "DisableScan": "false",
```

Comment:
opposite for any other env, we want to be scanning by default in all of our envs.